error: lint `clippy::safe_fn` has been renamed to `clippy::safe_code`
   |
   = help: use the new name `clippy::safe_code`
   = note: requested on the command line with `-D clippy::safe_fn`
   = note: `-D renamed-and-removed-lints` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(renamed_and_removed_lints)]`

error: lint `clippy::safe_fn` has been renamed to `clippy::safe_code`
   |
   = help: use the new name `clippy::safe_code`
   = note: requested on the command line with `-D clippy::safe_fn`
   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`

error: item must be `pub`
  --> tests/ui/crashes/unreachable-array-or-slice.rs:1:1
   |
LL | struct Foo(isize, isize, isize, isize);
   | ^ help: make this item public: `pub`
   |
   = note: requested on the command line with `-D clippy::missing-pub`

error: function must be `unsafe`
  --> tests/ui/crashes/unreachable-array-or-slice.rs:3:5
   |
LL | pub fn main() {
   |     ^
   |
help: make this function unsafe: `unsafe`
  --> tests/ui/crashes/unreachable-array-or-slice.rs:3:5
   |
LL | pub fn main() {
   |     ^
   = note: `-D clippy::safe-code` implied by `-D clippy::safe-fn`
   = help: to override `-D clippy::safe-fn` add `#[allow(clippy::safe_code)]`

error: variable must be `mut`
  --> tests/ui/crashes/unreachable-array-or-slice.rs:4:38
   |
LL |     let Self::anything_here_kills_it(a, b, ..) = Foo(5, 5, 5, 5);
   |                                      ^ help: make this variable mutable: `mut`
   |
   = note: requested on the command line with `-D clippy::missing-mut`

error: variable must be `mut`
  --> tests/ui/crashes/unreachable-array-or-slice.rs:4:41
   |
LL |     let Self::anything_here_kills_it(a, b, ..) = Foo(5, 5, 5, 5);
   |                                         ^ help: make this variable mutable: `mut`

error[E0433]: failed to resolve: `Self` is only available in impls, traits, and type definitions
  --> tests/ui/crashes/unreachable-array-or-slice.rs:4:9
   |
LL |     let Self::anything_here_kills_it(a, b, ..) = Foo(5, 5, 5, 5);
   |         ^^^^ `Self` is only available in impls, traits, and type definitions

error: aborting due to 7 previous errors

For more information about this error, try `rustc --explain E0433`.
